src/gui/application.cpp

```diff
@@ -67,60 +67,32 @@ Q_LOGGING_CATEGORY(lcApplication, "gui.application", QtInfoMsg)
 
 bool Application::configVersionMigration()
 {
-    QStringList deleteKeys, ignoreKeys;
-    AccountManager::backwardMigrationSettingsKeys(&deleteKeys, &ignoreKeys);
-    FolderMan::backwardMigrationSettingsKeys(&deleteKeys, &ignoreKeys);
-
     ConfigFile configFile;
 
     // Did the client version change?
     // (The client version is adjusted further down)
-    const bool versionChanged = QVersionNumber::fromString(configFile.clientVersionWithBuildNumberString()) != OCC::Version::versionWithBuildNumber();
-
-    // We want to message the user either for destructive changes,
-    // or if we're ignoring something and the client version changed.
-    bool warningMessage = !deleteKeys.isEmpty() || (!ignoreKeys.isEmpty() && versionChanged);
+    auto configVersion = QVersionNumber::fromString(configFile.clientVersionWithBuildNumberString());
+    auto clientVersion = OCC::Version::versionWithBuildNumber();
 
-    if (!versionChanged && !warningMessage)
+    if (configVersion == clientVersion) {
+        // no migration needed
         return true;
+    }
 
-    const auto backupFile = configFile.backup();
-
-    if (warningMessage) {
-        QString boldMessage;
-        if (!deleteKeys.isEmpty()) {
-            boldMessage = tr("Continuing will mean <b>deleting these settings</b>.");
-        } else {
-            boldMessage = tr("Continuing will mean <b>ignoring these settings</b>.");
-        }
-
-        QMessageBox box(
-            QMessageBox::Warning,
-            Theme::instance()->appNameGUI(),
+    if (clientVersion > configVersion) {
```

Comment:
I guess we should limit this to the major version.
Currently we even include the build number.

Reply:
done